Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -13,15 +15,76 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         view.addSubview(mapView)
 
+        // Setup and create button for toggling accuracy ring
+        self.setupToggleShowAccuracyButton()
+
         // Granularly configure the location puck with a `Puck2DConfiguration`
         let configuration = Puck2DConfiguration(topImage: UIImage(named: "star"))
         mapView.location.options.puckType = .puck2D(configuration)
         mapView.location.options.puckBearingSource = .course
+
+        // Center map over the user's current location
+        mapView.mapboxMap.onNext(.mapLoaded, handler: { [weak self] _ in
+            guard let self = self else { return }
+
+            if let currentLocation = self.mapView.location.latestLocation {
+                let cameraOptions = CameraOptions(center: currentLocation.coordinate, zoom: 18.0)
+                self.mapView.camera.ease(to: cameraOptions, duration: 2.0)
+            }
+        })
+
+        // Setup a toggle button to show how to toggle the accuracy radius
+        mapView.mapboxMap.onEvery(.cameraChanged, handler: { [weak self] _ in
+            guard let self = self,
+                  let button = self.toggleAccuracyRadiusButton else {
+                return
+            }
+
+            if self.mapView.cameraState.zoom >= 18.0 {
+                button.isHidden = false
+
+            } else {
+                button.isHidden = true
+            }
```

Comment:
How could you rewrite this as a 1-liner that uses an expression instead of control flow?